src/tasks/tasks.controller.js

```diff
@@ -62,11 +64,11 @@ taskRouter.delete('/:id', async (req,res) => {
         const task = await deleteTask(parseInt(id));
         console.log('deleteTask: ' + task);
         if (!task) {
-            throw new Error;
+            handleError(errorCustom, res);
         }
-        return res.status(200).json(task);
+        return buildResponse(res, 204, task);
```

Comment:
now task === true, after you'll made changes in repo in delete method task will become equal id. 
You can call buildResponse with string = `Task with id: ${id} deleted successful` instead task